potato/authentificaton.py

```diff
@@ -0,0 +1,79 @@
+import os
+import json
+
+class UserAuthenticator:
```

Comment:
I think this is a good idea to store users and auth in memory as most instances of this object won't have to hold too many users. We don't have to support a database.